db/controllers/api.controller.js

```diff
@@ -0,0 +1,7 @@
+const endpoints = require("../../endpoints.json")
```

Comment:
You have the endpoints here in an object, you could just send this in your controller